### ISSUE TITLE
Users need to be able to delete migration plans

### ISSUE THREAD

**Issue by serenamarie125** [ManageIQ/manageiq-v2v#298]
Users need to be able to delete migration plans

BEFORE STARTING WORK - need to find out the following:
- what restrictions are there around deleting a migration plan?  FYI - what states are delete available for?  Just Completed & Not Started ?
- need a design for this from @vconzola showing 1) where will delete action be available 
2) text for delete confirmation
3) what if you are trying to delete something that cannot be deleted, how do we information the user? 
4) what if the deletion is unsuccessful, how do we deal with that ?

API support already exists.  Ping Bill Wei with any questions.

**Comment by vconzola** [comment]
@serenamarie125 Is this issue superceded by the one for archiving a migration plan? Can this issue be closed?

**Comment by vconzola** [comment]
@serenamarie125 Can we close this one?

**Comment by serenamarie125** [comment]
@vconzola this is a future looking Issue - i think there was still talk about deleting in the future, isn't there ?

**Comment by vconzola** [comment]
@serenamarie125 Oh, I didn't read closely enough. I thought this was a duplicate of deleting mappings. My B.  Not sure if we'll ever delete plans, but doesn't hurt to keep the story open in case we do.

**Comment by vconzola** [comment]
Mockups:
![screen shot 2018-08-31 at 10 58 42 am](https://user-images.githubusercontent.com/22823035/45053530-3b1a7380-b058-11e8-9a4a-414d67294dad.png)
![screen shot 2018-08-31 at 10 58 20 am](https://user-images.githubusercontent.com/22823035/45053549-45d50880-b058-11e8-87e1-461fc91321cd.png)

**Comment by mturley** [comment]
Perfect, thanks @vconzola

**Comment by mturley** [comment]
For my own reference: according to https://docs.google.com/document/d/1B-iFdNKqUgw44GPHc0TIUEDc9MdKiBll2pK6PTve__M/edit?ts=5b898be4, plan deletion is available in the following states: `Not Started`, `Complete - Succeed`, `Complete - Fail`, `Archived`.

**Comment by vconzola** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1638508
